src/hotspot/share/runtime/globals.hpp

```diff
@@ -642,6 +642,10 @@ const intx ObjectAlignmentInBytes = 8;
   product(bool, OmitStackTraceInFastThrow, true,                            \
           "Omit backtraces for some 'hot' exceptions in optimized code")    \
                                                                             \
+  product(bool, OptimizeImplicitExceptions, true,                           \
```

Comment:
Should it be a diagnostic flag? Regular product flags require a CSR.

Reply:
Good point. Changed as suggested.